mmv1/templates/terraform/examples/target_https_proxy_certificate_manager_certificate.tf.erb

```diff
@@ -0,0 +1,46 @@
+
+resource "google_compute_target_https_proxy" "<%= ctx[:primary_resource_id] %>" {
+  name             = "<%= ctx[:vars]['target_https_proxy_name'] %>"
+  url_map          = google_compute_url_map.default.id
+  certificate_manager_certificates =  ["//certificatemanager.googleapis.com/${google_certificate_manager_certificate.default.id}"] # [google_certificate_manager_certificate.default.id] is also acceptable
```

Comment:
I'm a little unclear why a user would want to use the first format. Isn't it always exactly the same, except longer, and it requires string interpolation if a `google_certificate_manager_certificate` is being used? As I read through this PR, I couldn't help but wonder if a single format would be sufficient (which would also simplify a few pieces).

Reply:
In both cases, I convert the input to a full URL format `"//certificatemanager.googleapis.com/...."`. I have put the longer one to show how the full URL can be constructed in TF. 

There is another field in this resource (`certificateMap`) that expects a full url format, so I thought maybe customers will think of providing the resource in a full URL to keep the consistency (i.e. all certificates manager references are provided in a full URL). On the other hand, I wanted also to provide some easier option for a better UX (which is just using the `id`). What do you think?

Reply:
Makes sense, thanks for explaining!